src/app/static/src/app/store/translations/locales.ts

```diff
@@ -462,6 +436,50 @@ const fr: Partial<Translations> = {
+    axeHowHeading: "Dans quelle mesure le site Web est-il accessible",
+    axeHowContent: "Certaines parties de ce site Web peuvent ne pas être entièrement accessibles. Par exemple:",
+    axeHowExampleList: "<li>Il peut ne pas être entièrement navigable au clavier</li>\n" +
+        "                    <li>Certains graphiques et tableaux peuvent ne pas être correctement interprétés par les lecteurs d'écran</li>\n" +
+        "                    <li>Certains éléments de formulaire peuvent manquer de noms ou d'étiquettes descriptives</li>" +
+        "                    <li>Certains phrases peuvent ne pas avoir été correctement traduites</li>\n" +
+        "                    <li>Certains éléments de formulaire peuvent manquer de contrastes de couleurs suffisants</li>" +
+        "                    <li>Certains pages peuvent ne pas avoir la possibilité de sauter la navigation et d'accéder au contenu</li>",
+    axeWhatHeading: "Ce que nous faisons face aux problèmes connus",
+    axeWhatContent: "Nous travaillons pour atteindre et maintenir les <a href=\"https://www.w3.org/TR/WCAG21/\"> normes WCAG 2.1 AA </a>,\n" +
+        "                    mais ce n'est pas toujours possible pour tous nos\n" +
+        "                    contenu pour être accessible. Lorsque le contenu n'est pas accessible, nous indiquerons une raison, avertirons les utilisateurs et\n" +
+        "                    proposer des alternatives.",
+    axeTechnicalHeading: "Informations techniques sur l'accessibilité de ce site Web",
+    axeTechnicalParagraph1: "Nous nous engageons à rendre son site Web accessible conformément aux\n" +
+        "                    Organismes sectoriels (sites Web et applications mobiles) (No. 2) Règlement sur l'accessibilité 2018.",
+    axeTechnicalParagraph2: "Ce site Web est partiellement conforme au <a href=\"https://www.w3.org/TR/WCAG21/\"> contenu Web\n" +
+        "                    Directives d'accessibilité version 2.1</a> Norme AA, en raison des problèmes connus énumérés ci-dessus.",
+    axeReportingHeading: "Signaler des problèmes d'accessibilité",
+    axeReportingParagraph1: "Si vous avez besoin d'informations sur ce site Web dans un format différent, tel que PDF accessible, gros caractères, facile\n" +
+        "                    lecture, enregistrement audio ou braille ou si vous trouvez des problèmes d'accessibilité non répertoriés sur cette page\n" +
+        "                    veuillez contacter <a href=\"mailto:reside@imperial.ac.uk\"> reside@imperial.ac.uk </a>.",
+    axeReportingParagraph2: "Nous examinerons votre demande et vous répondrons dans les 7 jours.",
+    axeEnforcementHeading: "Procédure d'exécution",
+    axeEnforcementParagraph: "La Commission pour l'égalité et les droits de l'homme (EHRC) est chargée de faire respecter les organes du secteur public\n" +
+        "                    (Sites Web et applications mobiles) (n ° 2) Règlement sur l'accessibilité 2018 (le\n" +
```

Comment:
Better change this `(n ° 2)` too.

Also, this still has the suspicious singular `Règlement` x 2 (it's plural in the English) - see suggested change in previous review, or get a proper French speaker to override!

Reply:
Rich has looked at the content and I have made changes where he suggested.